### PR TITLE
Remove recovery from patient disease status

### PR DESCRIPTION
**Pull request by divyagar**
Please check [this](https://github.com/coronasafe/care_fe/issues/988) issue for information.

### CODE REVIEW

File: care/facility/models/patient_base.py [hunk 1]
```diff
@@ -72,9 +72,8 @@ class DiseaseStatusEnum(enum.IntEnum):
     SUSPECTED = 1
     POSITIVE = 2
     NEGATIVE = 3
-    RECOVERY = 4
-    RECOVERED = 5
-    EXPIRED = 6
+    RECOVERED = 4
+    EXPIRED = 5
```
Comment (vigneshhari) [inline care/facility/models/patient_base.py:76]: Never use an old choice integer for something else, use another integer

File: care/facility/management/commands/clean_patient_disease_status.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            try:
+                patient.disease_status = new
+                patient.save()
+            except Exception:
+                failed.append({"id": patient.id, "disease_status": patient.disease_status})
+
+            print(patient.disease_status)
+    return failed
+
+
```
Comment (vigneshhari) [inline care/facility/management/commands/clean_patient_disease_status.py:20]: we can just do patient.filter(disease_status=5).update(disease_status=4) right ?

**Review by vigneshhari — COMMENTED** [review]
Changes Required

File: care/facility/management/commands/clean_patient_disease_status.py (hunk 2 above)
Comment (divyagar) [reply care/facility/management/commands/clean_patient_disease_status.py:20]: > we can just do patient.filter(disease_status=5).update(disease_status=4) right ?

Yes we can but then we will not know updation of which rows are failed.

Comment (vigneshhari) [reply care/facility/management/commands/clean_patient_disease_status.py:20]: there will be one single mysql query, what failure are we talking about here ?

**Review by vigneshhari — CHANGES_REQUESTED** [review]
Changes Requested

**Comment by abhiabhi94** [comment]
Hey @vigneshhari, I think this change to 'RECOVERY' to `RECOVERED` should ideally be handled by a custom migration. This way, we won't necessarily have to run the command at every end(production as well as for all developers).

If you agree, I can do that and then this one only needs to handle the addition of another value to the model.

**Comment by vigneshhari** [comment]
@abhiabhi94 you can take over this issue then, @divyagar after he is done with his work review it.

**Comment by vigneshhari** [comment]
@abhiabhi94 close this PR when you start working on the issue

**Comment by abhiabhi94** [comment]
Do we need to add an extra field `EXPIRED`?

**Comment by vigneshhari** [comment]
https://github.com/coronasafe/care_fe/issues/988
This is the issue

**Pull request closed without merging**